api/swagger.yml

```diff
@@ -529,6 +529,15 @@ paths:
             type: integer
             minimum: 1
             maximum: 2049
+        - name: includePendingL1s
+          in: query
+          required: false
+          description: |
+            If set to true L1 transactions that have been addeed to the smart contract queue but haven't been forgrd yet are returned.
+            Warning: the correctness of the order is not guaranteed when using this filter, as the unforged transactions may change their position
```

Comment:
Does this mean that during pagination, you can get inconsistent results if this flag is true?  For example, in the first page you get the txs as pending, and in the second pagination you get them again as forged (in the case they are forged between the first and second page query)?